Volume/Views/Orgs Login/OrgsLoginView.swift

```diff
@@ -0,0 +1,180 @@
+                .background(
+                    RoundedRectangle(cornerRadius: 4)
+                        .strokeBorder(
+                            viewModel.showErrorMessage ? Color.volume.errorRed :
+                                slugIsFocused ? Color.volume.orange : Constants.textFieldBorderColor, style: StrokeStyle(lineWidth: 1)
+                        )
+                        .background(slugIsFocused ? Color.volume.offWhite : nil)
+                )
+                .focused($slugIsFocused)
+        }
+        .foregroundColor(Constants.textColor)
+    }
+    
+    private var codeInput: some View {
+        VStack(alignment: .leading, spacing: Constants.labelSpacing) {
+            Text("Access Code")
+                .font(Constants.labelFont)
+            
+            SecureField("Enter your access code", text: $viewModel.accessCode)
+                .font(Constants.textFieldFont)
+                .padding(Constants.textFieldPadding)
+                .background(
+                    RoundedRectangle(cornerRadius: 4)
+                        .strokeBorder(
+                            viewModel.showErrorMessage ? Color.volume.errorRed :
+                                accessCodeIsFocused ? Color.volume.orange : Constants.textFieldBorderColor, style: StrokeStyle(lineWidth: 1)
+                        )
+                        .background(accessCodeIsFocused ? Color.volume.offWhite : nil)
+                )
+                .focused($accessCodeIsFocused)
+                .keyboardType(.numberPad)
+                .onReceive(Just(viewModel.slug)) { _ in
+                    if viewModel.accessCode.count > Constants.maxAccessCodeLength {
+                        viewModel.accessCode = String(viewModel.accessCode.prefix(Constants.maxAccessCodeLength))
+                    }
+                }
+        }
+        .foregroundColor(Constants.textColor)
+    }
+    
+    private var authenticateButton: some View {
+        Button {
+            Task {
+                await viewModel.authenticate(accessCode: viewModel.accessCode, slug: viewModel.slug)
+            }
+        } label: {
+            Text("Authenticate")
+                .font(Constants.buttonTextFont)
+                .foregroundColor(viewModel.buttonEnabled ? Color.volume.offWhite : Constants.textColor)
+                .padding(.vertical, 12)
+                .frame(maxWidth: .infinity)
+                .background(
+                    RoundedRectangle(cornerRadius: 4)
+                        .foregroundColor(viewModel.buttonEnabled ? Color.volume.orange : Color.volume.buttonDisabledGray)
+                )
+        }
+        .padding(.top, 8)
+        .disabled(!viewModel.buttonEnabled)
+    }
+    
+    private var errorMessage: some View {
+        HStack(alignment: .center,spacing: 4) {
+            Image.volume.error
+                .frame(width: 16, height: 16)
+            
+            Text("Invalid slug or code. Please try again.")
```

Comment:
I think we decided on telling them if slug is invalid but giving an ambiguous error message if the access code is wrong. Double check with design (Liam did this screen)

Reply:
As mentioned in my PR description above. "AFAIK, there isn’t yet a way to see if it’s the slug or the access code that is invalid."

Reply:
Update: I think I figured it out. Will do this later.